src/app/components/modules/TopRightMenu.scss

```diff
@@ -1,12 +1,18 @@
 .sub-menu {
     li {
-        padding: .7rem 0.5rem;
         a {
             padding: 0px;
         }
     }
     li:last-child {
-        padding: 0 0 0 1rem;
+    }
+}
+
+.Header__usermenu {
+    li {
+        a {
+            padding: 0.7rem 1rem;
+        }
```

Comment:
master:

![screenshot from 2018-01-25 12-30-15](https://user-images.githubusercontent.com/99194/35402795-9dfe5ebc-01cb-11e8-84e4-946114d7d14c.png)


this branch:

![screenshot from 2018-01-25 12-25-12](https://user-images.githubusercontent.com/99194/35402801-a23b9b70-01cb-11e8-8cbe-232ffcbe7e2b.png)

Reply:
Will fix.